src/multiselect.js

```diff
@@ -116,6 +116,23 @@ export class Multiselect {
     }
   }
 
+  /**
+   * Ignore multi-field updates within the given function.
+   * @param {function} func The function to call.
+   */
+  withoutMultiFieldUpdates(func) {
+    let oldGroup = Blockly.Events.getGroup();
+    // Note that this depends on the fact that eventListener_ will ignore events with a group ID.
+    if (!oldGroup) {
+      Blockly.Events.setGroup("ignoreMultiFieldUpdates");
+    }
+    try {
+        func();
+    } finally {
+      Blockly.Events.setGroup(oldGroup);
+    }
+  }
+
```

Comment:
I would still recommend moving this to `src/index.js` and getting the function exported directly, since anyway we are not using any private methods or fields of the `MultiSelect` class, so this is more like a static method.

Reply:
How about if I just make it static, then?  One reason why I'd like to keep it as part of the Multiselect class is for plugin users (like App Inventor) who don't load the plugin via an  ES6 `import`.  In those use cases the exported names are just added to the top-level, `window` object, and so in this case would just look something like `window.withoutMultiFieldUpdates` without any indication that is is related to the Multiselect plugin. Adding it to the `Multiselect` class makes that association clear.

Reply:
Sounds good